src/main/java/com/airlines/model/CrewMember.java

```diff
@@ -100,11 +112,11 @@ public boolean equals(Object o) {
             return false;
         }
         CrewMember that = (CrewMember) o;
-        return getId() == that.getId() && Objects.equals(getFirstName(), that.getFirstName()) && Objects.equals(getLastName(), that.getLastName()) && getPosition() == that.getPosition() && Objects.equals(getBirthday(), that.getBirthday());
+        return getId() == that.getId() && getCrewId() == that.getCrewId() && Objects.equals(getFirstName(), that.getFirstName()) && Objects.equals(getLastName(), that.getLastName()) && getPosition() == that.getPosition() && Objects.equals(getBirthday(), that.getBirthday());
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(getId(), getFirstName(), getLastName(), getPosition(), getBirthday());
+        return Objects.hash(getId(), getCrewId(), getFirstName(), getLastName(), getPosition(), getBirthday());
```

Comment:
I don't think crewId should be part of hashcode or equals. It's discussable of course and depends on business logic, but I just wanted to mention that no strong need for this, but it's all up to you. You could leave it

Reply:
done in https://github.com/pielena/airlines-jdbc-app/commit/21a3eec6b8bb439efba0fe86562c5f31b291e47e